src/data/markdown/docs/03 cloud/04 Integrations/06 Grafana App.md

```diff
@@ -5,90 +5,25 @@ excerpt: 'Use k6 Cloud in Grafana'
 
 <Blockquote mod="attention" title="">
 
-This plugin is no longer maintained, and will be replaced by a new one soon. Stay tuned.
+The [k6 Cloud Grafana App plugin](https://grafana.com/grafana/plugins/grafana-k6-app/) is deprecated.
```

Comment:
This link is going to break as we're removing the plugin from the listing. We should probably remove this link.